core/data-plane/data-plane-util/src/main/java/org/eclipse/edc/connector/dataplane/util/sink/OutputStreamDataSinkFactory.java

```diff
@@ -38,9 +38,21 @@ public boolean canHandle(DataFlowRequest request) {
 
     @Override
     public @NotNull Result<Boolean> validate(DataFlowRequest request) {
-        return canHandle(request) ? Result.success(true) :
+        return validateRequest(request).succeeded() ? Result.success(true) :
                 Result.failure(String.format("%s: Cannot handle destination data address with type: %s",
                         getClass().getSimpleName(), request.getDestinationDataAddress().getType()));
```

Comment:
I think this could be simplified to:
`return validateRequest(request).map(it -> true);`